CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 [Unreleased]: https://github.com/chaostoolkit/chaostoolkit-kubernetes/compare/0.26.2...HEAD
 
+## [0.26.3][] - 2022-04-05
+
+### Fixed
+
+-Fixed the cmd splitting issue, therefore the shell commands will now parse correctly
```

Comment:
Ideally, if there is an open issue for this, you want to link it here.

Reply:
i don't see an issue that is the same as this, should I open one?

Reply:
You can, yes. This is usually the right approach :)
Then you can link it (look at other linked issues in the file for the syntax).